src/rpcblockchain.cpp

```diff
@@ -1079,10 +1085,7 @@ UniValue advertisebeacon(const UniValue& params, bool fHelp)
         res.pushKV("FAILURE","Note: if your wallet is locked this command will fail; to solve that unlock the wallet: 'walletpassphrase <yourpassword> <240>'.");
 
     else
-    {
```

Comment:
Please do not remove curly even when there is only one statement left. Error prone.

Reply:
That is complete BS Tomas. :| Then you should go and change all of the if (fDebug) Logprintf foo and all sorts of other areas of the code to 

if (fDebug)
{
    LogPrintf (foo)
}

Reply:
And Goddammit, markdown keeps changing the code format.

Reply:
Anyway. Apologize for getting annoyed. Seems a real nit-pick.

Reply:
I suppose if we were to follow Bitcoin's style guide the letter, what comes after an else clause should always be bracketed.

Reply:
There's always `-Wmisleading-indentation`.

Reply:
I will put the curly braces back in and merge the PR. We have spent way too much time on a trivial PR I think....

Reply:
I apologise.

Reply:
No problem Tomas. :)